docs/install_with_docker.md

```diff
@@ -0,0 +1,47 @@
+# Install BladeDISC with Docker
+
+[Docker](https://www.docker.com/) is a light container system, it helps
+users to package software and isolate BladeDISC runtime environment from
+the rest of the system. BladeDISC CI system released BladeDISC with different
+tag on [Docker Hub](https://hub.docker.com/repository/docker/bladedisc/bladedisc/tags?page=1&ordering=last_updated)
+repository.
+
+Nvidia Docker is an easy way to use GPU on Linux, please go to
+[install guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+to install it on your host.
+
+## Download a BladeDISC Docker Image
+
+BladeDISC released TensorFlow and PyTorch wrappers in separate Docker images
+on [bladedisc/bladedisc](https://hub.docker.com/repository/docker/bladedisc/bladedisc/tags?page=1&ordering=last_updated).
+The released Image tag is as the following table:
+
+| Docker tag | Description |
+| -- | -- |
+| latest-runtime-torch1.7.1 | latest release of BladeDISC, includes PyTorch 1.7.1 and CUDA 11.0 |
+| latest-runtime-tensorflow1.15 | latest release of BladeDISC, includes TensorFlow 1.15 and CUDA 11.0 |
+| latest-runtime-tensorflow2.24 | latest release of BladeDISC, includes TensorFlow 1.15 and CUDA 11.0 |
+| latest-devel-cuda10.0 | latest build of development environment, includes CUDA 11.0 and required development toolkit |
+| latest-devel-cuda11.0 | latest build of development environment, includes CUDA 11.0 and required development toolkit |
```

Comment:
Not all tags are available at `https://hub.docker.com/r/bladedisc/bladedisc/tags`.